distros/generate.sh

```diff
@@ -9,7 +9,13 @@ set -e
 #	or: ./generate.sh debian-newversion
 #		to create a new folder and a Dockerfile within it
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+# `readlink -f` does not work on macOS.
+# To get `greadlink`, run `brew install coreutils`.
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
Do you mean `gnu seq`? If so, yes. Do you prefer that? I preferred the other because it was cross-platform. I again don't have a strong preference, so I will follow your preference.